sql.txt

```diff
@@ -1,14 +1,18 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
-
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country ASC LIMIT 10 OFFSET 10;
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
-
+SELECT * FROM Customers WHERE ContactName LIKE "A%";
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
-
+SELECT * FROM OrderDetails WHERE ProductID = 51 AND Quantity > 10;
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
-
+INSERT INTO Products VALUES (null, "Calming Lavendar Tea", 3, 1, "10 boxes of 25 bags", 45);
+INSERT INTO Products VALUES (null, "Canada's Maple", 1, 2, "1 case of 50 bottles", 200);
+INSERT INTO Products VALUES (null, "Alaskan Cat Food", 19, 8, "30 - 2 oz cans", 28);
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
-
+UPDATE Products SET Price = (Price + 1.50) WHERE ProductID = 1;
+UPDATE Products SET Price = (Price + 1.50)  WHERE ProductID = 2;
```

Comment:
Similar to the comment above, this works, but try and do it in just one statement!